### ISSUE TITLE
DrawQQplot can draw indistiguishible points

### ISSUE THREAD

**Issue by mbaudin47** [openturns/openturns#1357]
Consider the following script.
```
import openturns as ot

distribution = ot.Normal()

sample = distribution.getSample(1000)
sample[0,0] = -5. # Outlier

ot.VisualTest_DrawQQplot(sample,distribution)
```

This produces :

![QQplot-bad](https://user-images.githubusercontent.com/31351465/72208997-0ddea080-34aa-11ea-8435-48a5345310cd.png)

The outlier cannot be seen in the graphics. Here why: the code switches between several point styles depending on the sample size. In our code, this amounts to the "dot" point style. 

https://github.com/openturns/openturns/blob/12c832da852833d12a9a9a428641f5b82b624735/lib/src/Uncertainty/StatTests/VisualTest.cxx#L59

The only way to get a consistent behavior is:

```
graph = ot.VisualTest_DrawQQplot(sample,distribution)
cloud = graph.getDrawable(1)
cloud.setPointStyle("bullet")
graph.setDrawable(cloud,1)
```

which produces:

![QQplot-good](https://user-images.githubusercontent.com/31351465/72209020-7299fb00-34aa-11ea-9da2-5edb673d077a.png)

This way, we can see the outlier point. 

I understand that the goal was to be able to manage a large sample size. However, the current implementation is unsatisfactory. 

My option would be to constantly use the PointStyle("bullet"), which is large enough in practice.

**Comment by regislebrun** [comment]
You are right, the thresholds (100 and 1000) should not be hard-coded but rather taken from ResourceMap, in the spirit of the other algorithms/objects in OT

**Comment by mbaudin47** [comment]
I agree, but I do not think that this solves the issue, because the points would still be near to invisible. Would'nt it be simpler to use the same point size whatever the sample size ?

**Comment by jschueller** [comment]
#3281
